Makefile

```diff
@@ -23,12 +23,12 @@ OS := $(shell go env GOOS)
 ARCH := $(shell go env GOARCH)
 
 # Kind
-KIND_VERSION := 0.9.0
+KIND_VERSION := 0.12.0
 KIND := ${BIN}/kind-${KIND_VERSION}
 K8S_CLUSTER_NAME := sample-external-issuer-e2e
 
 # cert-manager
-CERT_MANAGER_VERSION ?= 1.3.0
+CERT_MANAGER_VERSION ?= 1.7.1
```

Comment:
drive-by: 1.7.2 was released yday, might be worth using that instead :ok_hand:

Reply:
I'll upgrade to the 1.8 alpha when it's ready, in a separate PR.